source/_patterns/02-organisms/components/highlight.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
We should have variants with 1 and 2 items also. To achieve this we should not use `highlight.json` and instead just encourage the use of variants `highlight~full.json`, `highlight~one-item.json` and `highlight~two-items.json`. We do not need to support more than 3.

Reply:
@chris-huggins can you provide a design for one and two items for all screens

Reply:
@viktoria-mahmud as less than 3 highlights will just be on view on the preview instance of journal then it's not so important to get the design perfect.

Let's just see how they appear when there is one or two highlights and then we can either just agree that is acceptable or suggest a minor adjustment to make it not look terrible.